components/suggestion/command_provider.jsx

```diff
@@ -23,7 +23,7 @@ class CommandSuggestion extends Suggestion {
                 {...Suggestion.baseProps}
             >
                 <div className='command__title'>
-                    <string>{item.suggestion} {item.hint}</string>
+                    {`${item.suggestion} ${item.hint}`}
```

Comment:
Nit:`<Span>` would be more appropriate than to have template literal inside div

Reply:
Sure.
How about the `{item.description}` below? Should I wrap it with `<span>` as well? Thanks

Reply:
Done update, please take a look again, thanks